.goreleaser.yaml

```diff
@@ -208,7 +255,6 @@ nfpms:
     # Defaults empty, which means no filtering.
     builds:
       - linux-amd64
-      - linux-arm64
```

Comment:
We actually haven't build the GUI for ARM because pyinstaller doesn't support cross compilation so we can't automate that in GitHub actions at the moment. So this actually makes clear the fact that we were not building the GUI on arm.